testsuite/features/profiles/mgr.suse.de/Docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM registry.mgr.suse.de/toaster-sles12sp3-products
```

Comment:
Warning just in case, apparently we use sles12sp3 image, but then in the line number 13, we add sles12sp4 repo, all right?

Reply:
yes, all this is fake. We just want to see if we can build a container. We do not care if it is running.
When there is time for refactoring, we can think about creating special containers for this test and special repos.
I do not like so much, that we use `toaster` images.